src/components/Booking.jsx

```diff
@@ -24,27 +25,63 @@ const StyledBreak = styled.div`
   border: 1px;
   border-bottom-style: solid;
   border-bottom-color: #EBEBEB;
+  margin-top: 24px;
+  margin-bottom: 24px;
 `;
 
 class Booking extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      numViews: 0
+      numViews: 0,
+      isDatesOpen: false,
+      isGuestsOpen: false,
+      numAdults: 0,
+      numChildren: 0,
+      numInfants: 0,
     };
   }
 
+  toggleGuestsDropdown() {
+    this.setState({ isGuestsOpen: !this.state.isGuestsOpen });
+  }
+
+  updateNumAdults(value) {
+    this.setState(prev => {
+      return {
+        numAdults: prev.numAdults += value
+      };
+    });
+  }
+
+  updateNumChildren(value) {
+    this.setState(prev => {
+      return {
+        numChildren: prev.numChildren += value
+      };
+    });
+  }
+
+  updateNumInfants(value) {
+    this.setState(prev => {
+      return {
+        numInfants: prev.numInfants += value
+      };
+    });
+  }
+
   render() {
     return <StyledBooking>
       <CloseButton />
       <Price />
       <Reviews />
       <StyledBreak />
       <Dates />
-      <Guests />
+      <Guests isOpen={this.state.isGuestsOpen} toggle={this.toggleGuestsDropdown.bind(this)} numAdults={this.state.numAdults} numChildren={this.state.numChildren} numInfants={this.state.numInfants} updateNumAdults={this.updateNumAdults.bind(this)} updateNumChildren={this.updateNumChildren.bind(this)} updateNumInfants={this.updateNumInfants.bind(this)}/>
```

Comment:
This is something nit picky but In the future when you have time maybe refactor to airbnb standards. basically object destructure the states/ props. Makes it easier to read.